### PR TITLE
Adds consistency to accessibility support and assumptions wording

### LINKED ISSUE #932: inconsistent default text for Assumptions and Accessibility Support
The default text for Assumptions and Accessibility Support varies: sometimes with italics, sometimes with full-stop at the end, and combinations thereof. It seems the template changed over time. Also the template (wrongly) does have a full-stop at the end of the text, and also needs clean-up.

Note: The two texts/sentences are written in different style editorially and could be refined in the process.

### PR DESCRIPTION
**Pull request by daniel-montalvo**
For assumptions:

"No assumptions".

For accessibility support:

"No accessibility support issues known".

Closes issue(s):
- closes #932 (ADD ISSUE NUMBER HERE)

Need for Call for Review:
This will require a 1 week Call for Review << small changes affecting a small number of test cases, if in doubt do not use this. >>

---

### **When creating PR:**

- [x] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [x] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [x] Add yourself (and co-authors) as "Assignees" for PR.
- [x] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [x] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [x] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [x] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/aria-attr-defined-5f99a7.md [hunk 1]
```diff
@@ -24,11 +24,11 @@ Each target attribute is defined in [WAI-ARIA Specifications][].
 
 ## Assumptions
 
-_There are currently no assumptions_
+No assumptions.
```
Comment (WilcoFiers) [inline _rules/aria-attr-defined-5f99a7.md]:
```suggestion
There are no assumptions.
```

File: _rules/aria-attr-defined-5f99a7.md [hunk 2]
```diff
@@ -24,11 +24,11 @@ Each target attribute is defined in [WAI-ARIA Specifications][].
...
 
 ## Accessibility Support
 
-There are no major accessibility support issues known for this rule.
+No accessibility support issues known.
```
Comment (WilcoFiers) [inline _rules/aria-attr-defined-5f99a7.md]:
```suggestion
There are no accessibility support issues known.
```

File: _rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ When followed, the links in each set of target elements resolve to the [same res
 
 ## Accessibility Support
 
-_There are no major accessibility support issues known for this rule._
+_There are There are no accessibility support issues known.._
```
Comment (Jym77) [inline _rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md]: There are several with a double dot at the end.
```suggestion
There are no accessibility support issues known.
```

File: _rules/sequentially-focusable-element-has-visible-focus-oj04fd.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ This rule assumes that documents with only one focusable element automatically s
 
 ## Accessibility Support
 
-There are no major accessibility support issues known for this rule.
+There are There are no accessibility support issues known..
```
Comment (Jym77) [inline _rules/sequentially-focusable-element-has-visible-focus-oj04fd.md]: Also not sure why some (like here) are in straight up txt and some (like [this one](https://github.com/act-rules/act-rules.github.io/pull/1945/files#diff-579ce0c887300ad57b274db4b0c63bfbcdbb8e74c9aaebdf54ad0b326fbb8cd3R63)) are in italics/emphasised text.

**Review by Jym77 — DISMISSED** [review]
Double dots need to be cleaned.
We probably want to unify italics or no italics for these texts.

File: _rules/sequentially-focusable-element-has-visible-focus-oj04fd.md (hunk 4 above)
Comment (giacomo-petri) [reply _rules/sequentially-focusable-element-has-visible-focus-oj04fd.md]: > Also not sure why some (like here) are in straight up txt and some (like [this one](https://github.com/act-rules/act-rules.github.io/pull/1945/files#diff-579ce0c887300ad57b274db4b0c63bfbcdbb8e74c9aaebdf54ad0b326fbb8cd3R63)) are in italics/emphasised text.

@Jym77, considering I've "recently" created a new rule and I've remembered the italics/emphasised behaviour, I think it's coming from the rule template here https://act-rules.github.io/pages/design/rule-template/

Might be worth to also update it.

Comment (daniel-montalvo) [reply _rules/sequentially-focusable-element-has-visible-focus-oj04fd.md]: @giacomo-petri  I'd say I included updates to those files as well. Feel free to raise it if I did not.

Comment (WilcoFiers) [reply _rules/sequentially-focusable-element-has-visible-focus-oj04fd.md]: I'd like for them to be italic personally. I think that sets them apart from the ones where there is information. Slightly easier to scan the page that way.

**Review by WilcoFiers — APPROVED** [review]
For the next persons reviewing this. The example changes in this come from Prettier running automatically. These should not impact the implementations.

**Review by tombrunet — APPROVED** [review]

**Review by ajanec01 — APPROVED** [review]

**Pull request merged**